sky/skylet/providers/aws/node_provider.py

```diff
@@ -737,7 +740,9 @@ def get_command_runner(
             "use_internal_ip": use_internal_ip,
         }
         if docker_config and docker_config["container_name"] != "":
-            return SkyDockerCommandRunner(docker_config, **common_args)
+            return SkyDockerCommandRunner(
+                docker_config, self.docker_login_config, **common_args
```

Comment:
nit: is it possible to have the login_config directly included in the `docker_config` instead? Otherwise, it is a bit weird to have two arguments for docker related stuff.